### PR TITLE
Steamed Hams, but it's a Metasploit Module

### PR DESCRIPTION
**Pull request by busterb**
While similar to `exploit/multi/handler`, this module also provides an [unforgettable luncheon experience](https://www.youtube.com/watch?v=mkX3dO6KN54). It's also a handy method of synchronizing your exploit window with your buddies, a-la [Hudson Hawk](https://www.youtube.com/watch?v=D8KvM3vZo0w). Multiple targets and payloads are supported.

## Verification Steps

  1. Start msfconsole
  2. Do: ```use exploit/multi/steamed_hams```
  3. Do: ```set target [0 or 1]```
  4. Do: ```exploit```
  5. Enjoy!

:steam_locomotive: :hamburger: :hamburger: :hamburger:

### CODE REVIEW

**Review by sempervictus — COMMENTED** [review]
A fine example of why reviews are important... if this gives you root on my machine, will you fix the random IRQ storms? ;-)

File: modules/exploits/multi/steamed_hams.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+VWgtIEF1cm9yYSBib3JlYWxpcwpDaGFsbWVyczogYXQgdGhpcyB0aW1lIG9mIHllYXIKQ2hhbG1l
+cnM6IGF0IHRoaXMgdGltZSBvZiBkYXkKQ2hhbG1lcnM6IGluIHRoaXMgcGFydCBvZiB0aGUgY291
+bnRyeQpDaGFsbWVyczogbG9jYWxpemVkIGVudGlyZWx5IHdpdGhpbiB5b3VyIGtpdGNoZW4/ClNr
+aW5uZXI6IFllcy4KQ2hhbG1lcnM6IE1heSBJIHNlZSBpdD8KU2tpbm5lcjogTm8uCk1vdGhlcjog
+U2V5bW91ciEKTW90aGVyOiBUaGUgaG91c2UgaXMgb24gZmlyZSEKU2tpbm5lcjogTm8sIE1vdGhl
+ci4gSXQncyBqdXN0IHRoZSBub3J0aGVybiBsaWdodHMuCkNoYWxtZXJzOiBXZWxsLCBTZXltb3Vy
+LCB5b3UgYXJlIGFuIG9kZCBmZWxsb3cgYnV0IEkgbXVzdCBzYXkgeW91IHN0ZWFtIGEgZ29vZCBo
+YW0uCk1vdGhlcjogSGVscCEKTW90aGVyOiBIZWxwIQpGaXJldHJ1Y2s6IHdoZWVlcnJycgo=)
+        } ],
+        [ 'Legitmate Theater', {
```
Comment (wvu-r7) [inline modules/exploits/multi/steamed_hams.rb]: Legit, mate? Legitimate?

Comment (busterb) [reply modules/exploits/multi/steamed_hams.rb]: Legit, thanks.

**Comment by busterb** [comment]
This is expected to fail the sanity check, for reasons that may be self-evident!

**Comment by Chiggins** [comment]
Ship it!

**Comment by jmartin-r7** [comment]
Jenkins test this please.

**Comment by wvu-r7** [comment]
y u close?

**Comment by Auxilus** [comment]
😁😅

**Comment by busterb** [comment]
## Release Notes

Added exploit module for regional dialects.

**Comment by wvu-r7** [comment]
:meat_on_bone:

**Pull request merged**